packaging/osx/datadog-agent

```diff
@@ -89,6 +90,14 @@ case $1 in
         else
             echo "$DESC was not running"
         fi
+        checks=0
```

Comment:
Could you add a comment explaining what you're doing in this block?

Also, it makes sense to wait 10 seconds because by default that's the time supervisor will wait for the child processes to stop gracefully before it force-kills them with a `SIGKILL` signal (see `stopwaitsecs` option in http://supervisord.org/configuration.html#program-x-section-values). Let's explain that in the comment :)